packages/react-composites/tests/browser/call/hermetic/HorizontalGallery.test.ts

```diff
@@ -73,4 +74,117 @@ test.describe('HorizontalGallery tests', async () => {
       'horizontal-gallery-with-many-audio-participants-on-page-1.png'
     );
   });
+
+  test('HorizontalGallery should have 1 joining participant in the horizontal gallery', async ({ page, serverUrl }) => {
+    test.skip(isTestProfileStableFlavor());
+
+    const paul = defaultMockRemoteParticipant('Paul Bridges');
+    addVideoStream(paul, true);
+    const vasily = defaultMockRemoteParticipant('Vasily Podkolzin');
+    vasily.state = 'Connecting';
+
+    const participants = [paul, vasily];
+    const initialState = defaultMockCallAdapterState(participants);
+
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState));
+
+    await waitForSelector(page, dataUiId(IDS.videoGallery));
+    expect(await stableScreenshot(page, { dismissTooltips: true })).toMatchSnapshot(
+      'horizontal-gallery-with-joining-participant.png'
+    );
+  });
+
+  test('HorizontalGallery should have multiple audio participants and 1 calling participant', async ({
+    page,
+    serverUrl
+  }) => {
+    test.skip(isTestProfileStableFlavor());
+
+    const paul = defaultMockRemoteParticipant('Paul Bridges');
```

Comment:
Let's see the snapshots for this, but it looks like there is only 1 audio participant here (contrary to the test title)